assembly/pentaho-server/src/main/descriptors/assembly.xml

```diff
@@ -7,48 +7,48 @@
   <baseDirectory>pentaho-server</baseDirectory>
   <files>
     <file>
-      <source>${project.root.build.directory}/${project.artifactId}/pentaho-solutions/system/dialects/hsqldb/applicationContext-spring-security-hibernate.properties</source>
+      <source>${project.root.build.directory}/pentaho-server/pentaho-solutions/system/dialects/hsqldb/applicationContext-spring-security-hibernate.properties</source>
```

Comment:
Ugh... misread the change ... actually read it the other way around... yep, this PR is good by me 👍